Sources/App/AppDelegate.swift

```diff
@@ -260,6 +262,13 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         IntentHandlerFactory.handler(for: intent)
     }
 
+    func application(
+        _ application: UIApplication,
+        supportedInterfaceOrientationsFor window: UIWindow?
```

Comment:
you should not need to do this. use UIViewController-based interface orientation config.